### DISCUSSION TITLE
Forbid instances hosted on free TLD or shared domains like no-ip

### DISCUSSION THREAD

**Discussion by unixfox** [searxng/searx-instances#225] [Ideas]
Some instances like https://serx.ml, https://trydex.tk/searxng/, https://de.xcxc.ml/, https://saber.tk/ are registered under Freenom registrar: https://www.freenom.com which is a registrar for registering a domain with a TLD for free.

Usually websites under a free TLD are ephemeral, short-lived, mainly because the owner of the website didn't want to spend any money to start with.

Also in regard to freenom, there is an annoying policy that only allow you to renew your free TLD 2 weeks before its expiration, making it more likely to "forgot" to renew it, [see here](https://my.freenom.com/knowledgebase.php?action=displayarticle&id=2).
Moreover, if you forget to renew your freenom domain your domain is gone, you lost control over the domain.
In fact, freenom is the owner of the domain not the real owner of the website, [see here for more](https://hostingrevelations.com/is-freenom-safe/). This had led to hostile take over from freenom over existing domains, [see here](https://www.reddit.com/r/webhosting/comments/hxzsxo/is_freenom_legit/).

Based on these facts, what do you think of forbid any instances hosted on free TLD or shared domains like https://www.noip.com? We can kindly redirect the owner of the instance to buy a cheap real domain by finding a cheap registrar on https://tld-list.com for example.

Ping @dalf @ononoki1 @return42 @mrpaulblack @tiekoetter

**Comment by mrpaulblack** [comment]
Done in https://github.com/searxng/searx-instances/pull/226
